WorkloadManagementSystem/Utilities/test/Test_PilotCStoJSONSynchronizer.py

```diff
@@ -92,6 +96,25 @@ def setUp(self):
         }
       }
     }
+    Resources
```

Comment:
@fstagni: I had to add these sections for getCSDict to properly work, which was otherwise "silently" failing.
I now also added a return value check when getCSDict is called, and changes to return errors upwards.

Reply:
OK thanks. I didn't notice it.